components/UploadedFilePreview.js

```diff
@@ -183,9 +192,9 @@ UploadedFilePreview.propTypes = {
   isPrivate: PropTypes.bool,
   isLoading: PropTypes.bool,
   isDownloading: PropTypes.bool,
-  showFileName: PropTypes.bool,
   alt: PropTypes.string,
   fileName: PropTypes.string,
+  describe: PropTypes.string,
```

Comment:
`describe` as a variable name doesn't seem very explicit to me. We could rename it, but altogether I don't think it's the responsibility of this component to know about expenses and how they should be internationalized.

Could we rely on `fileName` and let the parents define the filename as they see fit?

Reply:
You definitely have a good point, but the `fileName` itself is used elsewhere in `UploadedFilePreview` so a new prop is still needed. Taking your suggestion elsewhere, I changed its name to `description` which is more indicative of what it actually is. Its default is `undefined`, so those calls to this function that don't render a description can preserve that behavior. But those that do will first construct an (internationalized) description and then pass it, where it will be rendered inside a `Box`.